### PR TITLE
Adding a basic LES duplicator

### PR DESCRIPTION
**Pull request by grdw**
Fixes: #1179

### CODE REVIEW

File: app/services/testing_ground/duplicator.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class TestingGround::Duplicator
```
Comment (antw) [inline app/services/testing_ground/duplicator.rb]: This class feels a *lot* like [`TestingGround::SaveAs`](https://github.com/quintel/etmoses/blob/master/app/services/testing_ground/save_as.rb) to me. Did you see that class? 😀

Comment (grdw) [inline app/services/testing_ground/duplicator.rb]: > Did you see that class?

Well no I didn't 👍 thanks for pointing that out.

**Comment by ThomBuijs** [comment]
I was looking for the status of #1179 and was referred to this issue. Is it closed and fixed?

**Comment by antw** [comment]
> Is it closed and fixed?

It is.

screen shot 2016-08-11 at 18 08 47 means that the change has been accepted, and is published to the "beta" server automatically within a couple of minutes.

**Comment by antw** [comment]
screen shot 2016-08-11 at 18 11 25

You may only clone a LES which is "public" (anyone can view it). If you have a LES which isn't public, you can create a clone by choosing "Edit LES", clicking the "General" tab near the top of the page, and then "Save as new Local Energy Solution".

**Pull request merged**